pkg/packer/manifests/vsphere/packer.json.tmpl

```diff
@@ -29,10 +29,10 @@
     "ssh_bastion_password": "{{ user `ssh_bastion_password`}}",
     "ssh_bastion_private_key_file": "{{ user `ssh_bastion_private_key_file` }}",
     "ssh_agent_auth": "false",
-    "ssh_private_key_file": "",
-    "ssh_password": "",
+    "ssh_private_key_file": "{{env `SSH_PRIVATE_KEY_FILE`}}",
```

Comment:
This will still let us set these with `ssh_username`?

Reply:
yes.